dsgrid/dataset/dataset_schema_handler_base.py

```diff
@@ -248,8 +252,20 @@ def _remap_dimension_columns(self, df, pivoted_columns=None):
         return df
 
     @track_timing(timer_stats_collector)
-    def _convert_time_dimension(self, load_data_df):
+    def _add_time_zone(self, load_data_df):
+        geography_dim = self._config.get_dimension(DimensionType.GEOGRAPHY)
+        geo_records = geography_dim.get_records_dataframe()
+        geo_name = geography_dim.model.dimension_type.value
+        load_data_df = add_column_from_records(load_data_df, geo_records, geo_name, "time_zone")
+        return load_data_df
+
+    @track_timing(timer_stats_collector)
+    def _convert_time_dimension(self, load_data_df, load_data_lookup=None):
         # This needs to convert the time format as well as time zone (TODO).
         time_dim = self._config.get_dimension(DimensionType.TIME)
-        time_dim.convert_dataframe(load_data_df, self._project_time_dim)
+        load_data_df = time_dim.convert_dataframe(
+            df=load_data_df,
+            project_time_dim=self._project_time_dim,
+            df_meta=load_data_lookup,
```

Comment:
What happens with this if the schema is one_table? Why isn't there similar code being added to the dataset_schema_handler_one_table.py file?

Reply:
We don't have a one-table dataset to test yet until AEO growth rates get added to the test dataset, and we do not want to add codes that are not ready to be tested.